projects/gateway2/translator/listener/gateway_listener_translator.go

```diff
@@ -314,15 +314,32 @@ func buildTcpHost(
 		})
 	}
 
-	if resolvedRefs {
-		// Set ResolvedRefs condition to True
+	if !hasValidBackend {
+		// No valid backends, do not create TcpHost
```

Comment:
Is there a reason we don't set a condition about this case and report it as a status?

Reply:
i would assume this should be a ResolvedRef=false

Reply:
Resolved in commit https://github.com/solo-io/gloo/commit/281ace000966aa7f15d960ab73d60efc4fa1c189 as this var, and the associated logic, is no longer used.